src/libs/Localize/index.ts

```diff
@@ -100,49 +91,38 @@ function translate(desiredLanguage = CONST.LOCALES.DEFAULT, phraseKey, phrasePar
 
 /**
  * Uses the locale in this file updated by the Onyx subscriber.
- *
- * @param {String|Array} phrase
- * @param {Object} [variables]
- * @returns {String}
  */
-function translateLocal(phrase, variables) {
-    return translate(BaseLocaleListener.getPreferredLocale(), phrase, variables);
+function translateLocal<TKey extends TranslationPaths>(phrase: TKey, ...variables: PhraseParameters<Phrase<TKey>>) {
+    return translate(BaseLocaleListener.getPreferredLocale(), phrase, ...variables);
 }
 
 /**
  * Return translated string for given error.
- *
- * @param {String|Array} message
- * @returns {String}
  */
-function translateIfPhraseKey(message) {
-    if (_.isEmpty(message)) {
+function translateIfPhraseKey(message: string | [string, Record<string, unknown> & {isTranslated?: true}]): string {
+    if (!message || (Array.isArray(message) && message.length > 0)) {
```

Comment:
```suggestion
    if (!message || (Array.isArray(message) && message.length === 0)) {
```

Shouldn't this be checking for an empty array instead?

Reply:
You are right, fixed it